cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -111,8 +141,12 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+          volumeMounts:
+          - name: eventer-config-volume
+            mountMath: /etc/config
           command:
             - /pod_nanny
+            - --config-dir=/etc/config
             - --cpu=100m
```

Comment:
@kawych  Actually, this flag no longer exists in the new version, so this won't work. Please create a new PR to remove unused flags.

Reply:
@MaciekPytel 

It will work, I've tested it :) But it looks like I've submitted the code into incorrect branch: https://github.com/kubernetes/autoscaler/blob/addon-resizer-release-1/addon-resizer/nanny/main/pod_nanny.go
I think we just need to cherrypick this to 1.8 branch. I can do a PR later today/early on monday.

Reply:
Ack. We'll need image version bump here later anyway.